test/fake_3par_data.py

```diff
@@ -0,0 +1,376 @@
+CHAP_PASS_KEY = "HPQ-docker-CHAP-secret"
+
+FLASH_CACHE_ENABLED = 1
+FLASH_CACHE_DISABLED = 2
+
+# EXISTENT_PATH error code returned from hpe3parclient
+EXISTENT_PATH = 73
+
+VOLUME_ID = 'd03338a9-9115-48a3-8dfc-35cdfcdc15a7'
+SRC_CG_VOLUME_ID = 'bd21d11b-c765-4c68-896c-6b07f63cfcb6'
+CLONE_ID = 'd03338a9-9115-48a3-8dfc-000000000000'
+VOLUME_TYPE_ID_REPLICATED = 'be9181f1-4040-46f2-8298-e7532f2bf9db'
+VOLUME_TYPE_ID_DEDUP = 'd03338a9-9115-48a3-8dfc-11111111111'
+VOL_TYPE_ID_DEDUP_COMPRESS = 'd03338a9-9115-48a3-8dfc-33333333333'
+VOLUME_TYPE_ID_FLASH_CACHE = 'd03338a9-9115-48a3-8dfc-22222222222'
+VOLUME_NAME = 'volume-' + VOLUME_ID
+SRC_CG_VOLUME_NAME = 'volume-' + SRC_CG_VOLUME_ID
+VOLUME_NAME_3PAR = 'osv-0DM4qZEVSKON-DXN-NwVpw'
+SNAPSHOT_ID1 = '2f823bdc-e36e-4dc8-bd15-de1c7a28ff31'
+SNAPSHOT_NAME1 = 'snapshot-2f823bdc-e36e-4dc8-bd15-de1c7a28ff31'
+SNAPSHOT_ID2 = '8da7488a-7920-451a-ad18-0e41eca15d25'
+SNAPSHOT_NAME2 = 'snapshot-8da7488a-7920-451a-ad18-0e41eca15d25'
+VOLUME_3PAR_NAME = 'osv-0DM4qZEVSKON-DXN-NwVpw'
+SNAPSHOT_3PAR_NAME = 'oss-L4I73ONuTci9Fd4ceij-MQ'
+RCG_3PAR_NAME = 'rcg-0DM4qZEVSKON-DXN-N'
+GROUP_ID = '6044fedf-c889-4752-900f-2039d247a5df'
+CONSIS_GROUP_NAME = 'vvs-YET.38iJR1KQDyA50kel3w'
+SRC_CONSIS_GROUP_ID = '7d7dfa02-ac6e-48cb-96af-8a0cd3008d47'
+SRC_CONSIS_GROUP_NAME = 'vvs-fX36AqxuSMuWr4oM0wCNRw'
+CGSNAPSHOT_ID = 'e91c5ed5-daee-4e84-8724-1c9e31e7a1f2'
+CGSNAPSHOT_BASE_NAME = 'oss-6Rxe1druToSHJByeMeeh8g'
+CLIENT_ID = "12345"
+REPLICATION_CLIENT_ID = "54321"
+REPLICATION_BACKEND_ID = 'target'
+# fake host on the 3par
+FAKE_HOST = 'fakehost'
+FAKE_CINDER_HOST = 'fakehost@foo#' + HPE3PAR_CPG
+USER_ID = '2689d9a913974c008b1d859013f23607'
+PROJECT_ID = 'fac88235b9d64685a3530f73e490348f'
+VOLUME_ID_SNAP = '761fc5e5-5191-4ec7-aeba-33e36de44156'
+FAKE_DESC = 'test description name'
+FAKE_FC_PORTS = [{'portPos': {'node': 7, 'slot': 1, 'cardPort': 1},
+                  'type': 1,
+                  'portWWN': '0987654321234',
+                  'protocol': 1,
+                  'mode': 2,
+                  'linkState': 4},
+                 {'portPos': {'node': 6, 'slot': 1, 'cardPort': 1},
+                  'type': 1,
+                  'portWWN': '123456789000987',
+                  'protocol': 1,
+                  'mode': 2,
+                  'linkState': 4}]
+
+VVS_NAME = "myvvs"
+
+# Provisioning
+THIN = 'thin'
+FULL = 'full'
+DEDUP = 'dedup'
+
+FAKE_ISCSI_PORT = {'portPos': {'node': 8, 'slot': 1, 'cardPort': 1},
+                   'protocol': 2,
+                   'mode': 2,
+                   'IPAddr': '1.1.1.2',
+                   'iSCSIName': ('iqn.2000-05.com.3pardata:'
+                                 '21810002ac00383d'),
+                   'linkState': 4}
+volume = {'name': VOLUME_NAME,
+          'id': VOLUME_ID,
+          'display_name': 'Foo Volume',
+          'size': 2,
+          'host': FAKE_CINDER_HOST,
+          'provisioning': THIN,
+          'flash_cache': None,
+          'volume_type': None,
```

Comment:
Do we need volume_type and volume_type_id

Reply:
We don't actually. Will remove it.